server/cpho/views/indicators.py

```diff
@@ -401,6 +403,7 @@ def get_context_data(self, **kwargs):
             "metadata_submission_statuses": get_metadata_submission_statuses(
                 indicator
             ),
+            "alternate_periods": alternate_periods,
```

Comment:
The idea here was to provide links for HSO to add data to adjacent years/quarters. Seems better to have this and not need it. There's no easy alternative way for HSO to add data in other periods